apps/envConstants.js

```diff
@@ -39,7 +37,7 @@ module.exports = {
   CIRCLECI: process.env.CIRCLECI,
   DRONE: process.env.DRONE,
   CIRCLE_TEST_REPORTS: process.env.CIRCLE_TEST_REPORTS,
-  // If set, will turn on react hot loader and run the webpack dev server
+  // If set, run webpack dev server (with hot reloading)
   HOT: !!process.env.HOT,
```

Comment:
...not our change, but how does one turn off this bool? Setting HOT=0 still returns true, does this need to be `process.env.HOT !== '0'`?

I'm a little confused about our proposed behavior for the "not hot" case -- it seems like webpack dev server is still running when I set hot to 0 (incl. the proposed change above to actually make the HOT=0 flag do something). Is that desirable?

Reply:
Most of our env vars have the property "having them set to anything makes them true", and going back to this just returns HOT to working like our other env vars. DEV works this way, etc, and its pretty common for env vars. It basically is equivalent to a flag like --hot, where absence means the opposite.

I don't object to having the check be `HOT !== "0"`, but I didn't see anything about this variable that particularly called for that behavior, now that we don't default to enabling HOT (it was important when not-specified=true to have a way to specify false, now that not-specified=false, its ok to have anything-specified=true).